docs/guides/charts-overview.md

```diff
@@ -0,0 +1,86 @@
+# Sensitive Data Archive Helm Charts
```

Comment:
```suggestion
# Sensitive Data Archive (SDA) Helm Charts
```

Reply:
The abbreviation `SDA` and the full name `Sensitive Data Archive` are used interchangeably throughout the text. I think we can add the abbreviation `(SDA)` at the beginning of the text and then use the short name `SDA` afterwards unless we need to use the full name by purpose.